bin/ovn-k8s-watcher

```diff
@@ -51,6 +51,9 @@ def main():
     parser.add_argument('--openstack', action='store_true')
 
     # Handle OVS logging and daemonization arguments.
+    ovs.dirs.RUNDIR = "/var/run/openvswitch"
+    ovs.dirs.LOGDIR = "/var/log/openvswitch"
```

Comment:
as far as you know is there any distro that uses different paths? I've never seen any, but in case there are these paths should be made configurable.